skimage/transform/_geometric.py

```diff
@@ -514,6 +522,28 @@ def estimate(self, src, dst):
         success : bool
             True, if model estimation succeeds.
 
+        Examples
+        --------
+        >>> tform_matrix = EssentialMatrixTransform(rotation=np.eye(3), translation=np.array([0, 0, 1]))
```

Comment:
This looks similar to the above example. Let's just move the example in this `estimate` function in the class docstring. I feel like one expressive example should be enough for the class.